### PR TITLE
Add support for Rails 7.0

### PR DESCRIPTION
**Pull request by jdmurphy**
Adds support for Rails 7.0 to the gem.

### CODE REVIEW

File: devise-uncommon_password.gemspec [hunk 1]
```diff
@@ -18,8 +18,10 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 2.3.0"
 
-  s.add_dependency "rails", ">= 4.2", "< 6.2"
+  s.add_dependency "rails", ">= 4.2", "< 7.1"
   s.add_dependency "devise", ">= 3.5", "< 4.9"
 
   s.add_development_dependency "sqlite3", ">= 1.3.6", "< 1.5"
+  # Needed for tests to get Rails 7 compatibility
+  s.add_development_dependency "sprockets-rails"
```
Comment (jdmurphy) [inline devise-uncommon_password.gemspec:26]: Sprockets is optional in Rails 7 so I need this for tests to run.

**Comment by HCLarsen** [comment]
Sorry, but I've had to revert this PR. Something about it broke the test suite.

**Comment by jdmurphy** [comment]
@HCLarsen did it break the travis test suite? When running testing locally I did not run into issues.

**Comment by HCLarsen** [comment]
I stopped using Travis when they stopped offering free support fro OSS. I tested locally, and it wouldn't run. Had an error related to sprockets. I've since fixed it, and I will update it on RubyGems within 24 hours.

**Comment by jdmurphy** [comment]
Thanks for the assist. Not sure why it was failing for you but not on my machine.

**Comment by HCLarsen** [comment]
Thank you for reminding me of the need for Rails 7 support. It's been pushed to RubyGems now.

**Pull request merged**